website/source/guides/upgrade/upgrade-specific.html.md

```diff
@@ -15,6 +15,32 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Nomad 0.10.0
+
+### Deployments
+
+Nomad 0.10 enables rolling deployments for service jobs by default
+and adds a default update stanza when a service job is created or updated.
+This does not affect jobs with an update stanza.
+In pre-0.10.0 releases, service jobs without an update stanza were deployed
+with destructive updates by default. To regain this behavior and
+disable deployments, set `max_parallel` to 0. 
+For more information, see [`update` stanza][update].
+
+### Raft 3
+Nomad 0.10 defaults to Raft 3 which includes [Autopilot](/guides/operations/autopilot.html), 
+operator-friendly automatic cluster management. To enable autopilot, all servers
+in a Nomad cluster must be running with Raft protocol version 3 or later.
+
+If existing servers are still on Raft 1, users will need to set the
```

Comment:
I'd avoid users, maybe `operators` or `you`, or use passive voice like rest of paragraph.